### ISSUE TITLE
Distributed computing for the `cluster` workflow

### ISSUE THREAD

**Issue by apcamargo** [bbuchfink/diamond#678]
Is it possible to run the `cluster` workflow across multiple nodes using the [instructions described in the Wiki](https://github.com/bbuchfink/diamond/wiki/6.-Distributed-computing)?

**Comment by bbuchfink** [comment]
No, to cluster on multiple nodes some manual work is needed. I have added a small howto: https://github.com/bbuchfink/diamond/wiki/How-to-cluster-huge-datasets
